package-lock.json

```
@@ -1,13 +1,14 @@
 {
   "name": "cuttle",
-  "version": "4.6.2",
+  "version": "4.6.3",
```

Should bump to match package.json at least b/c this installed deps
```suggestion
  "version": "4.7.0",
```